docs/introduction/getting-started.md

```diff
@@ -49,16 +49,46 @@ kubectl create secret generic awssm-secret --from-file=./access-key --from-file=
 
 ### Create your first SecretStore
 
+Create a file 'basic-secret-store.yaml' using `nano` or your preferred editor by running the following command. 
+
+```
+nano basic-secret-store.yaml
```

Comment:
It would be better if we could create a file using the `echo` since that is more OS agnostic 🙂

Reply:
Makes sense, I'll make a correction.  😃

Reply:
Does this [commit](https://github.com/external-secrets/external-secrets/pull/2751/commits/6c160129d655df3a09d262d1f699626717007907) solve the problem?

Reply:
Ah, sorry I meant `cat` not `echo`... Then we don't have to use nano or any other editors.

``` bash
cat > basic-secret-store.yaml << EOF
...
EOF
```

Reply:
Hmm, I see where you stack. We don't seem to be able to use `include` with bash commands. Why don't we just ask users to create a file without specifying an editor like below?


### Create your first SecretStore

Create a file 'basic-secret-store.yaml' with the following content.

```yaml
{ % include 'basic-secret-store.yaml' % }
```

Apply the new file to a resource by filename.

```bash
kubectl apply -f basic-secret-store.yaml
```

### Create your first ExternalSecret

Create a file 'basic-external-secret.yaml' with the following content.

```yaml
{ % include 'basic-external-secret.yaml' % }
```

Apply the new file to a resource by filename.

```bash
kubectl apply -f basic-external-secret.yaml
```

Reply:
Also, please be ware that you need to commit with `--sign` flag!

Reply:
I committed the changes.
Thanks for the advice.

Reply:
Thank you for fixing it! It seems the DCO job still fails. Would you fix it?

Reply:
Sorry for the delay, I followed the instructions provided for correction, was the problem resolved?